diff -up krdc-21.04.3/rdp/CMakeLists.txt.orig krdc-21.04.3/rdp/CMakeLists.txt
--- krdc-21.04.3/rdp/CMakeLists.txt.orig	2021-07-05 21:21:04.000000000 +0200
+++ krdc-21.04.3/rdp/CMakeLists.txt	2021-07-23 15:05:33.484235818 +0200
@@ -1,6 +1,4 @@
-if(FREERDP_EXECUTABLE_FOUND)
-    set(HAVE_XFREERDP true)
-endif(FREERDP_EXECUTABLE_FOUND)
+set(HAVE_XFREERDP true)
 
 if(HAVE_XFREERDP)
     add_definitions(-DKDE_DEFAULT_DEBUG_AREA=5012)
